icu4c/source/common/unicode/ucnv.h

```diff
@@ -537,14 +539,28 @@ ucnv_openPackage(const char *packageName, const char *converterName, UErrorCode
  *  allocations because warning codes can be overridden by subsequent
  *  function calls.
  * @return pointer to the new clone
- * @stable ICU 2.0
+ * @deprecated ICU 71 Use ucnv_clone() instead.
  */
 U_CAPI UConverter * U_EXPORT2
 ucnv_safeClone(const UConverter *cnv,
                void             *stackBuffer,
                int32_t          *pBufferSize,
                UErrorCode       *status);
 
+#endif /* U_HIDE_DEPRECATED_API */
```

Comment:
Suggestion: In this header file, move ucnv_clone() above ucnv_safeClone(), and merge the two U_HIDE_DEPRECATED_API sections for safeClone and its size constant.